src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,27 +1,61 @@
 import { values, every, capitalize } from 'lodash';
 import React, { Component } from 'react';
+import Raven from 'raven-js';
+import moment from 'moment';
 
 /**
  * VA Facility Known Operational Hours
  */
 export default class LocationHours extends Component {
-  colonizeTime(time) {
-    const found = time.match(/(\d?\d)(\d\d)(\w\w)/);
-    return `${found[1]}:${found[2]}${found[3]}`;
+  convertHour(hour, day) {
+    const hours = hour.split('-');
+    const isValid = hours.every(time => moment(time, 'hmmA').isValid());
```

Comment:
You could get rid of the extra parsing iteration (here and the bottom of the function) by doing `const hours = hour.split('-').map(time => moment(time, 'hmmA'));` and using the array of moment objects on this line and in the later return statement.